### PR TITLE
Re-opening PR for Descriptive Titles

### LINKED ISSUE #34: Add the Title field to webpages so that tab names are more descriptive
**Previously,**

All Tabroom.com web instances appeared identical in tabbed browsing, each displaying the generic label `Tabroom.com`.

image

If you had several tabs open, this made it extremely difficult to select the right one. It also makes it hard to return to a closed tab via browser history, since the Title is indexed in most browser histories, but it's only indexing `Tabroom.com` instead of any useful information. So typically, users need to re-navigate the whole site instead of just typing "Glenbrooks" or something to get back to where they were.

**Now,**

Major Tabroom subpages have titles that reflect their purpose.

image

- Tournament splash/"invite" pages: `<Name of Tournament>`
- Results pages: `Results - <Event Name> - <Round Name> - <Tournament Name>`
- Pairings pages: `Pairings - <Event Name> - <Round Name> - <Tournament Name>`
- Entries pages: `Entries - <Event Name> - <Tournament Name>`
- Judges pages: `Judges - <Tournament Name>`

There's probably others that could use a descriptive name, but these are probably the biggest "bang for your buck" pages.

### PR DESCRIPTION
**Pull request by benjmor**
Closes #34 

Re-opening since I don't trust my comment to a closed PR (#35 ) to not languish in an email inbox somewhere.

TLDR: I think more granular titles for webpages will be a big UI win.

### CODE REVIEW

**Comment by palmertab** [comment]
I obviated this one by doing the same thing in a way that wouldn't drag performance down; no shade on you for not knowing the ins and outs of friggin Perl Mason and how it can be slowed in some ways.

**Comment by bmorrissirromb** [comment]
I get that Tabroom 3.0 is kind of in KTLO mode, but I don't think your changes really covered more than about 10% of my PR. I'll pick this back up on Tabroom 4.0.
